### PR TITLE
Gulp bootlint

### PR DESCRIPTION
**Pull request by nicolaisueper**
Please review gulp config L45 - 65 :)

### CODE REVIEW

File: gulp.config.js [hunk 1]
```diff
@@ -42,6 +42,26 @@ export const settings = {
...
+		reportFn(file, lint, isError, errorLocation) {
+			let message = (isError) ? 'ERROR! - ' : 'WARN! - ';
+			if (errorLocation) {
+				message += file.path + ' (line:' + (errorLocation.line + 1) + ', col:' + (errorLocation.column + 1) + ') [' + lint.id + '] ' + lint.message;
+			} else {
+				message += file.path + ': ' + lint.id + ' ' + lint.message;
+			}
+			console.log(message);
+		},
+		summaryReportFn(file, errorCount, warningCount) {
```
Comment (mischah) [inline gulp.config.js]: Do we need a custom report?

File: gulp.config.js [hunk 2]
```diff
@@ -42,6 +42,26 @@ export const settings = {
 			'Opera 12.1'
 		]
 	},
+	bootlint: {
+		stoponerror: true,
```
Comment (mischah) [inline gulp.config.js]: `stoponerror` should only set to `true` for production build.

File: gulp.config.js (hunk 1 above)
Comment (nicolaisueper) [inline gulp.config.js]: Yes, because it doesn't produce output without it. Can you test it without on your machine?

File: gulp.config.js (hunk 2 above)
Comment (nicolaisueper) [inline gulp.config.js]: see 5b0dc9d :)

File: gulp.config.js (hunk 1 above)
Comment (mischah) [inline gulp.config.js]: Wat? Interesting. Going to check this locally.

**Pull request merged**